README.md

```diff
@@ -26,21 +26,74 @@ following rules, which we consider useful.
+
+Security Champions must attend a thorough training. They need an overview of the complete security development lifecycle (SDLC), theoretical background, 
+practical knowledge, and the necessary soft skills to fulfill its role. Examplary trainings are: 
+<!--
+- [Fraunhofer IEM - Security Champion Training](https://www.iem.fraunhofer.de/de/academy/schulungsangebot/security-champion-training.html)
+- TODO
+-->
+
+### Security Champion Guild
+
+The Security Champion Guild is a guild as known from the [Spotify model](https://www.atlassian.com/agile/agile-at-scale/spotify). Security Champions participate to get feedback on security-related topics from outside their teams such as individual questions, second opinions, and code/concept reviews.
+
+The main task of the guild is to spread the Security Belts in the organization. The guild conducts belt assessments and actively improves the maturity model.
+
+All Security Champions and only these are part of the guild. Otherwise, the champions would not learn from each other and the quality of the guild's outcome is at risk as Security Champions have a dedicated training, which other developers have not. If a central security department exists within the company, then the guild and this department need to collaborate.
+
+## Rituals
+
+The Security Belts define a list of rituals that shall be held such that the goals of our model can be accomplished. 
+
+### Weekly Review
+
+The Security Champion Guild organizes a review every week with one representative of each team working on achieving a Security Belt. If the team does not have a Security Champion yet, then the representative does not have to be a Security Champion.
+
+The weekly is set up like a daily standup in Scrum, where everyone presents the progress of their implementation of the Security Belt activity they currently working on.
+
+This meeting allows teams to get early feedback and to give the Security Champion Guild transparency about the progress the teams are making with their Security Belts.
+
+### Belt Assessment
+
+There is an assessment for each security belt to ensure that the team has performed the required activities and, thus, has matured concerning software security. 
+
+The team has to request the assessment through the guild. Since the Security Belts are intended to empower a team and not only the Security Champion, the whole team has to attend to the assessment. The guild organizes the assessors. There should be at least two Security Champions that already achieved the Security Belt that the assessment is about. If there are not enough suitable assessors, the whole guild and additional security experts of the organization attend the assessment and function as assessors.
```

Comment:
```suggestion
The team has to request the assessment through the guild. Since the Security Belts are intended to empower a team and not only the Security Champion, the whole team has to attend the assessment. The guild organizes the assessors. There should be at least two Security Champions that already achieved the Security Belt that the assessment is about. If there are not enough suitable assessors, the whole guild and additional security experts of the organization attend the assessment and function as assessors.
```